File: u-projects/university-projects/Servicios Web/Taller3/ConsumirServicios.php
<?php
require_once('/lib/nusoap.php');

// Create a client for the service
$objClienteSOAP = new nusoap_client('http://mottz.info/ictac2015/ExponerServiciosWebICTAC.php?wsdl','wsdl');
$param = array('ID' => '1144052533', 'tipoID' => 'ID');

$resultado1 = $objClienteSOAP->call('ConsultarInfoUsuario', $param);
$resultado2 = $objClienteSOAP->call('ConsultarDescargaCertificado', $param);
$resultado3 = $objClienteSOAP->call('ConsultarCertificadoUsuario', $param);

    
echo '<br>';
print_r($resultado1);
echo '<br>';

echo '<br>';
print_r($resultado2);
echo '<br>';

echo '<br>';
print_r($resultado3);
echo '<br>';
     
?>

File: u-projects/university-projects/Servicios Web/Taller3/Taller3/Clients/Ruby/ConsumirServicios.rb
require 'savon'

# create a client for the service
client = Savon.client(wsdl: 'http://mottz.info/ictac2015/ExponerServiciosWebICTAC.php?wsdl')

client.operations

# Consultar Certificado Usuario
response = client.call(:consultar_certificado_usuario, message: { id: "1144052533", tipoID: "ID" })
puts response.doc

# Consultar Descarga Certificado
response = client.call(:consultar_descarga_certificado, message: { id: "1144052533", tipoID: "ID" })
puts response.doc

# Consultar Informacion Usuario
response = client.call(:consultar_info_usuario, message: { id: "1144052533", tipoID: "ID" })
puts response.doc